lib/billet_manager/installments_basis/services/get_customers.ex

```diff
@@ -0,0 +1,20 @@
+defmodule BilletManager.InstallmentsBasis.Services.GetCustomers do
+  @moduledoc """
+  Service that list customers
```

Comment:
Não tem a necessidade de falar que é um service.

`Responsible for get users from database`